package.json

```diff
@@ -1,36 +1,59 @@
 {
-  "name": "hubspot-local-dev-tools",
-  "version": "1.0.0",
-  "description": "Local development tools for working with HubSpot",
+  "name": "@hubspot/cli",
+  "version": "6.2.1",
+  "description": "The official CLI for developing on HubSpot",
+  "license": "Apache-2.0",
   "repository": "https://github.com/HubSpot/hubspot-cli",
-  "private": true,
+  "dependencies": {
+    "@hubspot/local-dev-lib": "^2.0.1",
+    "@hubspot/serverless-dev-runtime": "6.2.0",
+    "@hubspot/theme-preview-dev-server": "0.0.8",
+    "@hubspot/ui-extensions-dev-server": "0.8.33",
+    "archiver": "^7.0.1",
+    "chalk": "^4.1.2",
+    "chokidar": "^3.0.1",
+    "cli-cursor": "^3.1.0",
+    "cli-progress": "^3.11.2",
+    "express": "^4.17.1",
+    "findup-sync": "^4.0.0",
+    "fs-extra": "^8.1.0",
+    "inquirer": "8.2.0",
+    "jest": "^29.5.0",
+    "js-yaml": "^4.1.0",
+    "moment": "^2.29.1",
+    "open": "^7.0.3",
+    "p-queue": "^6.0.2",
+    "strip-ansi": "^5.2.0",
+    "table": "^6.6.0",
+    "tmp": "^0.2.1",
+    "update-notifier": "^5.1.0",
+    "yargs": "17.7.2"
+  },
   "devDependencies": {
-    "depcheck": "1.3.1",
+    "axios": "^1.7.2",
     "eslint": "^8.56.0",
     "husky": "^4.3.8",
-    "lerna": "^6.6.1",
     "lint-staged": "^10.5.4",
     "madge": "^8.0.0",
+    "mock-stdin": "^1.0.0",
     "prettier": "^1.19.1",
-    "sass": "~1.29.0"
+    "semver": "^7.6.3",
+    "typescript": "^5.6.2"
   },
-  "workspaces": [
-    "packages/*",
-    "acceptance-tests"
-  ],
   "scripts": {
-    "check-deps": "node bin/depcheck.js",
-    "lint": "eslint . && prettier --list-different packages/**/*.{js,json}",
-    "prettier:write": "prettier --write packages/**/*.{js,json}",
-    "publish-release": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish --conventional-graduate --exact || echo 'Error: New release can only be published on main branch' >&2",
-    "publish-prerelease": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish prerelease --preid beta --dist-tag next --exact || echo 'Error: New release can only be published on main branch' >&2",
+    "build": "rm -rf ./dist/ && tsc --rootDir . --outdir dist && yarn copy-files",
```

Comment:
Can we move the `rootDir` and `outdir` settings to the `tsconfig.json`?

Reply:
I can't remember exactly why, but we ran into an issue with doing that in local-dev-lib and inlining it here solved it. Will look into that